Packs/Slack/Integrations/SlackV3/SlackV3_test.py

```diff
@@ -3506,7 +3531,7 @@ def test_send_file_no_args_investigation(mocker):
 
     # Assert
     assert SlackV3.slack_send_request.call_count == 1
-    assert success_results[0] == 'File sent to Slack successfully.'
+    assert success_results[0]['HumanReadable'] == 'File sent to Slack successfully.'
```

Comment:
was it not CommandResults earlier, and now it is?

Reply:
It was `demisto.result` before, now it is `CommandResults` so I had to update the test to access the `HumanReadable` attribute of the new object